molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/ExacAnnotator.java

```diff
@@ -0,0 +1,148 @@
+	public static final String EXAC_EUR_MAF = "EXAC_EUR_MAF";
+	public static final String EXAC_EUR_MAF_LABEL = "EXACEURMAF";
+	public static final String EXAC_AA_MAF = "EXAC_AA_MAF";
+	public static final String EXAC_AA_MAF_LABEL = "EXACAAMAF";
+	public static final String EXAC_EA_MAF = "EXAC_EA_MAF";
+	public static final String EXAC_EA_MAF_LABEL = "EXACEAMAF";
+
+	public static final String EXAC_FILE_LOCATION_PROPERTY = "exac_location";
+	public static final String EXAC_TABIX_RESOURCE = "EXACTabixResource";
+	private static final Logger LOG = LoggerFactory.getLogger(CaddAnnotator.class);
+
+	@Autowired
+	private MolgenisSettings molgenisSettings;
+	@Autowired
+	private DataService dataService;
+	@Autowired
+	private Resources resources;
+
+	@Bean
+	public RepositoryAnnotator exac()
+	{
+		List<AttributeMetaData> attributes = new ArrayList<>();
+
+		DefaultAttributeMetaData exac_ac = new DefaultAttributeMetaData(EXAC_AC, FieldTypeEnum.DECIMAL)
+		.setDescription("Allele count in genotypes derived straight from the ExAC AC field").setLabel(EXAC_AC_LABEL);
+		
+		DefaultAttributeMetaData exac_gmaf = new DefaultAttributeMetaData(EXAC_GMAF, FieldTypeEnum.DECIMAL)
+				.setDescription("Global minor allele frequency derived straight from the ExAC GMAF field").setLabel(EXAC_GMAF_LABEL);
+
+		DefaultAttributeMetaData exac_aa_maf = new DefaultAttributeMetaData(EXAC_AA_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("African/African latino minor allele frequency derived straight from the ExAC AMR_MAF field").setLabel(EXAC_AA_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_amr_maf = new DefaultAttributeMetaData(EXAC_AMR_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("American latino minor allele frequency derived straight from the ExAC AMR_MAF field").setLabel(EXAC_AMR_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_afr_maf = new DefaultAttributeMetaData(EXAC_AFR_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("African minor allele frequency derived straight from the ExAC AFR_MAF field").setLabel(EXAC_AFR_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_asn_maf = new DefaultAttributeMetaData(EXAC_ASN_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("Asia minor allele frequency derived straight from the ExAC ASN_MAF field").setLabel(EXAC_ASN_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_eur_maf = new DefaultAttributeMetaData(EXAC_EUR_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("European minor allele frequency derived straight from the ExAC EUR_MAF field").setLabel(EXAC_EUR_MAF_LABEL);
+
+		DefaultAttributeMetaData exac_ea_maf = new DefaultAttributeMetaData(EXAC_EA_MAF, FieldTypeEnum.DECIMAL)
+				.setDescription("European American minor allele frequency derived straight from the ExAC AE_MAF field").setLabel(EXAC_EA_MAF_LABEL);
+
+		attributes.add(exac_gmaf);
+		attributes.add(exac_afr_maf);
+		attributes.add(exac_amr_maf);
+		attributes.add(exac_asn_maf);
+		attributes.add(exac_ea_maf);
+		attributes.add(exac_eur_maf);
+		attributes.add(exac_aa_maf);
+
```

Comment:
Do we really want all of it? previously the EXAC annotator only added one column...